w3c/did_w3c.go

```diff
@@ -0,0 +1,818 @@
+// Package w3c is a set of tools to work with Decentralized Identifiers (DIDs)
+// as described in the DID spec https://w3c.github.io/did-core/
+// Got from https://github.com/build-trust/did
```

Comment:
Files in this package should have copyright notice with license and original authors.